CONTRIBUTING.md

```diff
@@ -88,27 +88,26 @@ Sygnal follows the [Synapse code style].
 Many of the conventions are enforced by scripts which are run as part of the
 [continuous integration system](#continuous-integration-and-testing).
 
-To help check and fix adherence to the code style, you can run `scripts-dev/lint.sh`
-locally. You'll need Python 3.7 or later, and to install a number of tools:
+To help check and fix adherence to the code style, you can run `tox`
+locally. You'll need Python 3.7 or later, and a virtual environment configured and
+active:
 
-```
-# Install the dependencies
-pip install -U black flake8 isort mypy mypy-zope
+```bash
+# Activate the virtual environment
+source ./venv/bin/activate
 
-# Run the linter script
-./scripts-dev/lint.sh
+# Run the code style check
+tox -e check_codestyle
+
+# Run the types check
+tox -e check_types
 ```
 
 **Note that the script does not just test/check, but also reformats code, so you
 may wish to ensure any new code is committed first**.
```

Comment:
We should probably delete this too, since the `tox` commands all have nice `--check-only` flags on them, in contrast to the old `lint.sh`.

Otherwise, this looks good to go from me!